src/contract_wrappers/tokens/security_token_wrapper/common.ts

```diff
@@ -1162,6 +719,10 @@ export default class SecurityTokenWrapper extends ERC20TokenWrapper {
     return this.contractFactory.getModuleRegistryContract();
   };
 
+  public subscribeAsync!: Subscribe;
```

Comment:
Once the class is abstract, these declarations aren't required

Reply:
you're right